src/video.c

```diff
@@ -259,11 +259,139 @@ static void layerToggleExpected(void) {
 	REG_DISPCNT = MODE_0 | BG0_ON;
 }
 
+static char oamDelayInside = 0;
+
+static void oamDelayVcount(void) {
+	while (!(REG_DISPSTAT & LCDC_HBL_FLAG));
+	//REG_DISPCNT ^= BG0_ON;
```

Comment:
Any reason to leave the commented out code in?

Reply:
I don't really remember anymore, but most likely not. I'll remove it.

Reply:
It doesn't look removed to me.

Reply:
Oh, apparently I removed the other instance of this. Oops. Fixed for real now.